qualtran/bloqs/chemistry/df/double_factorization.py

```diff
@@ -120,6 +120,28 @@ def control_registers(self) -> Iterable[Register]:
             Register("succ_p", QBit()),
         )
 
+    @property
+    def alpha(self) -> float:
+        # TODO: correct if necessary, see https://github.com/quantumlib/Qualtran/issues/1247
+        return 1.0
```

Comment:
what happens if you raise NotImplemented here?

Reply:
We discussed; changing to raise exception here.